### DISCUSSION TITLE
What is the default sync polling interval for auto sync and how it can be reduced to 5 second

### DISCUSSION THREAD

**Discussion by shuknk8s** [argoproj/argo-cd#7129] [Q&A]
The sync is consuming lot of time in auto sync mode before launching sync and not acceptable for demo purpose.
Need  to know precisely how to modify the long polling interval for auto sync and set to an acceptable value like 5s. The FAQ is pointing to different issue and no clear instruction has been provided either in docs of faqs.

**Comment by alexmt** [comment]
Hello @shuknk8s , you can change timeout using `timeout.reconciliation` key of `argocd-cm` ConfigMap

https://github.com/argoproj/argo-cd/blob/a9f009cb10765d6871cbb8e17c5873f5bc0a4a95/docs/operator-manual/argocd-cm.yaml#L228-L231

**Comment by shuknk8s** [comment]
How do I update the existing configmap to reflect the updated value? The
query did not show any DATA inside the configmap and I am not comfortable
applying the argocd-cm manifest from github as we are in bacl to back demo
initiative. Please provide the instructions for updating this property
using helm or kustomize if applicable.

Here is what I see when I queries the cm definition

piVersion: v1
kind: ConfigMap
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |

{"apiVersion":"v1","kind":"ConfigMap","metadata":{"annotations":{},"labels":{"
app.kubernetes.io/name":"argocd-cm","app.kubernetes.io/part-of
":"argocd"},"name":"argocd-cm","namespace":"argocd"}}
  creationTimestamp: "2021-08-24T17:19:58Z"
  labels:
    app.kubernetes.io/name: argocd-cm
    app.kubernetes.io/part-of: argocd
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          .: {}
          f:kubectl.kubernetes.io/last-applied-configuration: {}
        f:labels:
          .: {}
          f:app.kubernetes.io/name: {}
          f:app.kubernetes.io/part-of: {}
    manager: kubectl-client-side-apply
    operation: Update
    time: "2021-08-24T17:19:58Z"
  name: argocd-cm
  namespace: argocd
  resourceVersion: "577810"
  uid: 31a7f01e-a53f-4a4f-8341-27fdce7b3109
